### ISSUE TITLE
Enchant library returns a 404

### ISSUE THREAD

**Issue by MattGeri** [mlocati/docker-php-extension-installer#1001]
### Version of install-php-extensions

2.6.1

### Error description

When trying to install the `enchant` library, the installer returns a 404. This is the error message:

```
#0 76.61 ### INSTALLING BUNDLED MODULE enchant ###
#0 77.04 curl: (22) The requested URL returned error: 404
```

The URL referenced in the source of install-php-extensions also returns a 404: https://github.com/AbiWord/enchant/releases/download/enchant-1-6-1/enchant-1.6.1.tar.gz

### Docker image

php:7.4.33-fpm

### Minimal Dockerfile

```Dockerfile
FROM php:7.4.33-fpm

ADD https://github.com/mlocati/docker-php-extension-installer/releases/latest/download/install-php-extensions /usr/local/bin/

# Install PHP mods.
RUN chmod +x /usr/local/bin/install-php-extensions && install-php-extensions gd exif xdebug apcu enchant opcache imagick mysqli pdo pdo_mysql zip
```

**Comment by mlocati** [comment]
Thanks for the report! I've just published version 2.6.2 which resolves this issue.

**Comment by MattGeri** [comment]
Amazing, thanks so much for your effort and a great tool!
